demos/pkcs11/pkcs11_demo_mechanisms_and_digests/CMakeLists.txt

```
@@ -0,0 +1,52 @@
+set( DEMO_NAME "pkcs11_demo_mechanisms_and_digests" )
+
+# Set path to corePKCS11 and it's third party libraries.
+set(COREPKCS11_LOCATION "${CMAKE_SOURCE_DIR}/libraries/standard/corePKCS11")
+set(3RDPARTY_LOCATION "${CMAKE_SOURCE_DIR}/libraries/3rdparty")
```

This seems to be unused in the file
```suggestion
set(3RDPARTY_LOCATION "${CMAKE_SOURCE_DIR}/libraries/3rdparty")
```

---

Oops, was leftover from older work